### PR TITLE
add ogg argument support

### PR DESCRIPTION
**Pull request by dromzeh**
yo

made this as for my own use case - completely up to you if you want to merge but might be useful for those who are looking for something lightweight (like this CLI) and want to export in another format?

tbh this could be added onto like `--output ogg` for example (i can change it if you would prefer lol)

Code_fXRdT2MYGh

### CODE REVIEW

**Comment by Escartem** [comment]
ty for this pull request, like said in the readme I wanted to add ogg support but never actually did it, I'll take a look and merge it soon :)

**Comment by dromzeh** [comment]
> ty for this pull request, like said in the readme I wanted to add ogg support but never actually did it, I'll take a look and merge it soon :)

yeah, no worries man

if you were looking to expand this support than just additional `ogg` i can rewrite it to `--output [format]` then create a map for `format` => `codec` (e.g `ogg` => `libvorbis`) ? whatever u prefer lol

**Comment by Escartem** [comment]
All formats won't necessarily be a ffmpeg lib but I guess ogg is good enough for now

File: extract.py [hunk 1]
```diff
@@ -253,13 +269,10 @@ def main():
...
-				if alone:
-					os.makedirs(path(f"temp/map/unmapped"), exist_ok=True)
-				else:
-					if len(new_files) > 0:
-						os.makedirs(path(f"temp/map/new_files/unmapped"), exist_ok=True)
-					if len(changed_files) > 0:
-						os.makedirs(path(f"temp/map/changed_files/unmapped"), exist_ok=True)
+				os.makedirs(path(f"temp/map/unmapped"), exist_ok=True)
+				if not alone:
+					os.makedirs(path(f"temp/map/new_files/unmapped"), exist_ok=True)
```
Comment (Escartem) [inline extract.py:274]: does it works changed like that ?

**Comment by Escartem** [comment]
looking pretty good overall, I'll do some tests and merge it when I have a bit of time

**Pull request merged**